forestserviceprototype/specialuseform/views.py

```diff
@@ -28,7 +28,7 @@ def submit(request, permit_id=None, template_name='specialuseform/submit.html'):
         'form': form, 'submit_text': submit_button_text
     })
 
-def submitted_permit(request, permit_id):
+def submitted_permit(request, permit_id, check_status=False):
```

Comment:
when we build out more of the coming back to this this page from not submitting it, it will be true so we con't want to show this alert.